api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessConstants.java

```diff
@@ -16,4 +16,7 @@
     public static final  String LOCATION_USER_PROPERTY_NAME = "locationUuid";
 
     public static final String REF_APP_LOCATION_USER_PROPERTY_NAME = "referenceapplication.locationUserPropertyName";
+    public static final String PERSONATTRIBUTETYPE_NAME = "LocationAttribute";
```

Comment:
What do you think about locationUuid rather than LocationAttribute?

Reply:
As its the Name of the Attribute it's fine I guess.

Reply:
Is it the name of the attribute or name of the location Uuid?

Reply:
Name of the AttributeType